Mathlib/LinearAlgebra/Matrix/Spectrum.lean

```diff
@@ -55,6 +57,26 @@ lemma mulVec_eigenvectorBasis (j : n) :
       congr(⇑$((isHermitian_iff_isSymmetric.1 hA).apply_eigenvectorBasis
         finrank_euclideanSpace ((Fintype.equivOfCardEq (Fintype.card_fin _)).symm j)))
 
+/-- Eigenvalues of a Hermitian Matrix, coerced, belong to the spectrum of the assoc.toEuclideanLin -/
+theorem ofReal_eigenvalue_mem_spectrum_toEuclideanLin (i : n) :
+    (RCLike.ofReal ∘ hA.eigenvalues) i ∈ spectrum 𝕜 (toEuclideanLin A) :=
+  LinearMap.IsSymmetric.hasEigenvalue_eigenvalues _ _ _ |>.mem_spectrum
```

Comment:
How do you feel about just inlining this in the proof of `eigenvalues_mem_spectrum_real` below? We can always get back easily with `spectrum.algebraMap_mem` and `spectrum_toEuclideanLin`. I just feel like we don't really need many results about `spectrum 𝕜 (toEuclideanLin A)`.

Reply:
This is totally fine...I feel a bit weird about *having* these coercion-involved results, so inlining them seems better. Not sure this is the correct intuition, but...trying to learn the right design mojo...

Reply:
Used apply, but I think it's probably ok.